sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/display/DisplayData.java

```diff
@@ -101,6 +125,10 @@ public String toString() {
     return builder.toString();
   }
 
+  private static String convertNamespace(Class<?> nsClass) {
```

Comment:
`namespaceOf(...)`? `classToNamespace`? (just thinking out loud, feel free to ignore if you don't think they're improvements).